packages/odyssey-react-mui/src/labs/Layout.tsx

```diff
@@ -59,6 +59,14 @@ const LayoutContent = styled("div", {
   "& + &": {
     marginBlockStart: odysseyDesignTokens.Spacing4,
   },
+
+  ["@media (max-width: 800px)"]: {
+    display: "block",
+
+    "& > div + div": {
```

Comment:
I'm not really a fan of these deeply nested selectors in a very general component like this. I've gotten myself in trouble a lot with things like this haha. Can you show me what problem we are looking to solve with this?

Reply:
I found a more elegant fix. We need a way to move the layout, but also the layout's children, into a single column at smaller screen sizes.

Reply:
I've investigated doing this and I don't think we can reliable make the assumption that we can move things to a single column. I think we should avoid making assumptions about responsive behavior for the time being

Reply:
If we are going to make these types of assumptions, I've worked up a solution using container queries already in another branch